<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>batchImportDialog</class>
 <widget class="QDialog" name="batchImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>717</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Import Games Wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget &gt;QWidget{
	/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45,56, 75, 255), stop:1 rgba(27, 33, 41, 255));*/
	background-color: rgba(45,56, 75, 255);
	border:none;
}
QStackedWidget{
	border:none;
}
QWidget{
	background-color:rgb(36,40,47);
	color:rgb(200,200,200)
}

QFrame{
	background-color: rgb(42,46,51);
border:1px solid;
border-color:rgb(33, 34, 43);}

QLabel{
color: rgb(243, 244, 245);
border:none;}

QPushButton{
background-color: rgb(43,45,49);
color:rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}

QPushButton:hover:!pressed
{
background-color: rgb(54, 55, 61);
}

QTextEdit{
background-color:rgb(130,133,143);
color: rgb(243, 244, 245);
border:none;}

QPlainTextEdit{
color: rgb(130,133,143);
border:none;}

QListWidget{
background-color:rgb(130,133,143);
color: rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}

QTableWidget{
background-color:rgb(130,133,143);
color: rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}

QToolButton{
background-color: rgb(43,45,49);
color:rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}
QToolButton:hover:!pressed
{
background-color: rgb(54, 55, 61);
}

QCheckBox{
background-color:none;
color: rgb(243, 244, 245);
border:none;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="swImportGames">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Root Path:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tbPath"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnSetFolder">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Set Folder</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Autofill from path:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tbFormat">
               <property name="text">
                <string>{path}/{creator}/{title}/{version}</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="useRegex">
               <property name="text">
                <string>Skip regexify</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="helpText">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Valid options: path, title, creator, version. All must be surrounded with braces, Example: {path}
{path} and {title} are required at the minimum
Use &quot;/&quot; for subfolders and &quot;-&quot; if game info is in the folder name. See examples below:
somegame-v1.0/game.exe
somegame/v1.0/game.exe
Autofill from path uses regex as it's way of parsing paths. You can fill in raw regex here or use the simplified capture groups (See '{title}' and it's siblings).</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="flags">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="cbCheckLocal">
               <property name="text">
                <string>Search for game information in the local metadata datbase (recommended)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbDownloadBanners">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Download Banners</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Download metadata from VNDB</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="moveImported">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Move to Hydrus95 Game folder (Can be set individually in next page)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ready to Import</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>This wizard is ready to import the games below. Please review the list to confirm that the correct games will be imported,and make any necessary changes. Change the game names in the Name column or remove items with the Delete key
Hover over the column headers for help text</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="twGames"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBack">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
